programs/programs/co/child_care_assistance/calculator.py

```diff
@@ -46,7 +52,7 @@ def household_eligible(self, e: Eligibility):
         gross_income = self.screen.calc_gross_income(frequency, ["all"], ["cashAssistance"])
         deductions = self.screen.calc_expenses(frequency, ["childSupport"])
         net_income = gross_income - deductions
-        fpl_percent = cccap_county_limits[county_name] / 100
+        fpl_percent = cccap_county_limits.get(county_name, 0) / 100
```

Comment:
when the google sheets referenced in this calculator module are not accessible, this generates an empty `dict` and this throws a key error. Switching to `.get` allows for a default value.